commands/insult.js

```diff
@@ -0,0 +1,51 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const config = require('../config.json');
+const requestHandler = require('axios').default;
+const axios = requestHandler.create({
+    timeout: 500,
+});
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('insult')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
        .setName("insult")
```